src/env-variables.ts

```diff
@@ -1,37 +1,27 @@
-function getEnvVariable(name: string): string {
-  const envVariable = process.env[name];
-  if (!envVariable) {
-    throw new Error(`${name} environment variable is not set`);
-  }
-  return envVariable;
```

Comment:
Out of curiosity, what's the reasoning behind this change?

Reply:
We originally added this as a DX feature, but we've gotten reports that it's more annoying than helpful. See https://github.com/workos/authkit-nextjs/issues/45